content/en/core/overview/translations.md

```diff
@@ -20,7 +20,7 @@ Note that "keys" in .properties files are referred to as `terms` in POEditor.
 ## Adding new languages
 New languages must be added and configured in several places:
 - *In GitHub*
-  - Create a new [ddocs/medic/_attachments/translations/messages-XX.properties](https://github.com/medic/cht-core/tree/master/ddocs/medic/_attachments/translations) file, replacing XX with the 2 or 3 letter language code.
+  - Create a new [api/resources/translations/messages-XX.properties](https://github.com/medic/cht-core/tree/master/api/resources/translations) file, replacing XX with the 2 or 3 letter language code.
```

Comment:
Reads fine the way it is, and I think most people would figure it out, but here's an alternate phrasing that might be more technically accurate.

```suggestion
  - Create a new `messages-XX.properties` file in the [`api/resources/translations`](https://github.com/medic/cht-core/tree/master/api/resources/translations) folder, replacing XX with the 2 or 3 letter language code.
```